sandbox/namespace_linux.go

```diff
@@ -167,6 +168,9 @@ func (n *networkNamespace) RemoveInterface(i *Interface) error {
 }
 
 func (n *networkNamespace) AddInterface(i *Interface) error {
+	suffix := fmt.Sprintf("%d", n.nextIfIndex)
```

Comment:
Since you are not using suffix anywhere else, have you considered:

i.DstName = fmt.Sprintf("%s%d", i.DstName, n.nextIfIndex)

If you think it is cleaner as it is now, it's fine too.